docs/website/docs/contributing/usage-data.md

```diff
@@ -1,11 +1,12 @@
-== Usage Data Collection
-With user approval, we collect pseudonymized user data that will help us improve odo.
+---
+title: Usage Data Collection
+sidebar_position: 11
+---
+With user approval, we collect [pseudonymized](https://en.wikipedia.org/wiki/Pseudonymization) user data that will help us improve odo.
 
-To learn more about what data is being collected and how to configure this collection, see link:{https://github.com/openshift/odo/blob/main/USAGE_DATA.adoc}[USAGE_DATA.adoc].
+To learn more about what data is being collected and how to configure this collection, see [here](https://github.com/openshift/odo/blob/main/USAGE_DATA.md).
```

Comment:
Wouldn't it help if there was only one document to maintain for usage data?

Reply:
Well, this doc is intended for developers, and USAGE.md for general users. I would prefer to keep them separate.

Reply:
I could move this doc to command-reference section under `odo telemetry` command, but since this command is supposedly hidden, I am unsure whether we should add it.

Reply:
Keep only one instance of the usage data doc, IMO. Command reference is not the right place. Since it's affecting users, putting it on odo.dev is right. It could live under `Getting Started` section.